python/ray/_private/usage/usage_lib.py

```diff
@@ -182,6 +193,106 @@ def put_cluster_metadata(gcs_client, num_retries) -> None:
     return metadata
 
 
+def get_cluster_status(gcs_client, num_retries) -> dict:
+    """Get the current status of this cluster."""
+    cluster_status = ray._private.utils.internal_kv_get_with_retry(
+        gcs_client,
+        ray.ray_constants.DEBUG_AUTOSCALING_STATUS,
+        namespace=None,
+        num_retries=num_retries,
+    )
+    if not cluster_status:
+        return {}
+
+    result = {}
+    to_GiB = 1 / 2 ** 30
+    cluster_status = json.loads(cluster_status.decode("utf-8"))
+    if (
+        "load_metrics_report" not in cluster_status
+        or "usage" not in cluster_status["load_metrics_report"]
+    ):
+        return {}
+
+    usage = cluster_status["load_metrics_report"]["usage"]
+    if "CPU" in usage:
+        result["num_cpus"] = usage["CPU"][1]
+    if "GPU" in usage:
+        result["num_gpus"] = usage["GPU"][1]
+    if "memory" in usage:
+        result["total_memory_gb"] = usage["memory"][1] * to_GiB
+    if "object_store_memory" in usage:
+        result["total_object_store_memory_gb"] = (
+            usage["object_store_memory"][1] * to_GiB
+        )
+    return result
+
+
+def get_cluster_config(cluster_config_file_path) -> dict:
```

Comment:
@DmitriGekhtman Could you review this file to make sure I read `~/ray_bootstrap_config.yaml` correctly?

Reply:
This part lgtm

Reply:
It would be better to rename the method and call the output of the method something other than "cluster config" because that's easy to confuse with the cluster config that is the input of this method.

Maybe it should be called "cluster_config_summary" and should be an instance of a dataclass ClusterConfigSummary (or something like that)

Reply:
Changed to dataclass and named it ClusterConfigToReport